bindings/gumjs/gumdukobject.c

```diff
@@ -76,26 +72,40 @@ _gumjs_duk_add_properties_to_class (duk_context * ctx,
 
     duk_push_string (ctx, entry->name);
     idx++;
-    // [ class proto propname ]
+    // [ proto propname ]
     if (entry->getter != NULL)
     {
       idx++;
       flags |= DUK_DEFPROP_HAVE_GETTER;
       duk_push_c_function (ctx, entry->getter, 0);
-      // [ class proto propname getter ]
+      // [ proto propname getter ]
     }
     if (entry->setter != NULL)
     {
       idx++;
       flags |= DUK_DEFPROP_HAVE_SETTER;
       duk_push_c_function (ctx, entry->setter, 1);
-      // [ class proto propname {getter} setter ]
+      // [ proto propname {getter} setter ]
     }
 
     duk_def_prop (ctx, -idx, flags);
-    // [ class proto ]
+    // [ proto ]
   }
 
+  duk_pop (ctx);
+  // []
+}
+
+void
+_gumjs_duk_add_properties_to_class (duk_context * ctx, gchar * classname,
+    const GumDukPropertyEntry * entries)
```

Comment:
Ok. Note that the function declaration style is different between .h and .c, and here we're using the wrong one.